src/validator/validator.ts

```diff
@@ -1,8 +1,16 @@
-import { z, ZodFirstPartySchemaTypes } from 'zod';
+import { RefinementCtx, z, ZodFirstPartySchemaTypes } from 'zod';
+import { oisSchema } from './ois';
```

Comment:
Tbh I would love to just re-use the airnode-validator package to avoid code duplication. Today I saw that Emanuel was applying a fix to the validator code and I thought to myself that I would need to also apply the same fix in Airkeeper 😓 

The thing is that functions in airnode-validator expect to receive an Airnode config.json file and I'm not sure what would happen if I try validating a file with extra fields. Maybe we need to expose functions in airnode-validator to validate parts of the config file like you are mentioning: `validateOis`, `validateNodeSettings`, etc.

I'd like to get @amarthadan opinion on this too.

Reply:
> Maybe we need to expose functions in airnode-validator to validate parts of the config file like you are mentioning: validateOis, validateNodeSettings, etc.

Ois schema is exported from validator and also used in https://github.com/api3dao/airnode/blob/master/packages/airnode-ois/src/index.ts#L11. Other schemas are exported as well, but they are not accessible that nicely. For example to import nodeSettingsSchema you have to do:

```
import {nodeSettingsSchema} from '@api3/airnode-validator/dist/src/config';
```

But this is a detail. I think you should be using the validations from Airnode.